specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ContainerApps.json

```diff
@@ -551,6 +610,141 @@
           "type": "boolean"
         }
       }
+    },
+    "CustomDomain": {
+      "description": "Custom Domain of a Container App",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Hostname.",
+          "type": "string"
+        },
+        "bindingType": {
+          "description": "Custom Domain binding type.",
+          "enum": [
+            "Disabled",
+            "SniEnabled",
+            "IpBasedEnabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "bindingType",
+            "modelAsString": false
+          }
+        },
+        "virtualIP": {
+          "description": "Virtual IP address assigned to the hostname if IP based binding is enabled.",
+          "type": "string"
+        },
+        "certificateName": {
+          "description": "Name of the Certificate to be bound to this hostname. Must exist in the Managed Environment.",
+          "type": "string"
+        },
+        "hostType": {
```

Comment:
There is no repository hostname for containerapps AFAIK.

Reply:
Yes. Confirmed it. Removing it.